haystack/nodes/retriever/dense.py

```diff
@@ -855,58 +849,50 @@ def __init__(
                 "This can be set when initializing the DocumentStore"
             )
 
-        self.infer_tokenizer_classes = infer_tokenizer_classes
-        tokenizers_default_classes = {
-            "query": "DPRQuestionEncoderTokenizer",
-            "passage": "DPRContextEncoderTokenizer",
-            "table": "DPRContextEncoderTokenizer",
+        tokenizers_default_classes: Dict[str, Type[PreTrainedTokenizer]] = {
+            "query": DPRQuestionEncoderTokenizerFast,
```

Comment:
Do we want to hard code fast tokenizers? Just asking...

Reply:
Also, I don't see the point of this dict. We can use the Tokenizer classes directly here, we even remove a few lines of code....